css-values-5/Overview.bs

```diff
@@ -945,8 +945,47 @@ Generating/Caching Random Values: the <<random-caching-options>> value</h4>
 		</pre>
 	</div>
 
+<!--
+████████ ██       ████████ ██     ██ ████████ ██    ██ ████████        ██████   ███████  ██     ██ ██    ██ ████████ ████ ██    ██  ██████  
+██       ██       ██       ███   ███ ██       ███   ██    ██          ██    ██ ██     ██ ██     ██ ███   ██    ██     ██  ███   ██ ██    ██ 
+██       ██       ██       ████ ████ ██       ████  ██    ██          ██       ██     ██ ██     ██ ████  ██    ██     ██  ████  ██ ██       
+██████   ██       ██████   ██ ███ ██ ██████   ██ ██ ██    ██          ██       ██     ██ ██     ██ ██ ██ ██    ██     ██  ██ ██ ██ ██   ████
+██       ██       ██       ██     ██ ██       ██  ████    ██          ██       ██     ██ ██     ██ ██  ████    ██     ██  ██  ████ ██    ██ 
+██       ██       ██       ██     ██ ██       ██   ███    ██          ██    ██ ██     ██ ██     ██ ██   ███    ██     ██  ██   ███ ██    ██ 
+████████ ████████ ████████ ██     ██ ████████ ██    ██    ██           ██████   ███████   ███████  ██    ██    ██    ████ ██    ██  ██████  
+-->
+
+<h2 id=element-counting>
+Counting Elements</h2>
+
+	Authors often want to apply element specific styles
+	based on the position of the element within the tree.
+
+	The ''sibling-index()'' and ''sibling-count()'' functions
+	(the <dfn export for=CSS>element counting functions</dfn>)
+	allow authors to take these factors into account in ''calc()'' expressions.
+
+	With these, authors can create things like staggered animation delays
+	or easily position elements on a circle.
+
+<h3 id=random>
+Getting the position of an element within its siblings: the ''sibling-index()'' function</h3>
+
+	The <dfn>sibling-index()</dfn> function resolves to the position
+	of an element within its group of <a>inclusive siblings</a>.
+
+	Similar to '':nth-child()'', the list of elements is <b>1-indexed</b>;
+	that is, the first child of an element has index 1.
+
+<h3 id=random>
+Getting the number of siblings an element is part of: the ''sibling-count()'' function</h3>
+
 
+	The <dfn>sibling-count()</dfn> function resolves to the total number of
+	<a>inclusive siblings</a> an element is part of. That is, it resolves to 
+	the total number of direct children the element’s parent element contains.
 
+Issue: Describe when it actually resolves. Computed Value? Used Value?
```

Comment:
Definitely resolves computed-value-time, no reason to wait any longer.

Reply:
Added in af5ce47